core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -1186,6 +1185,8 @@ class GroupCoordinator(val brokerId: Int,
           for (member <- group.allMemberMetadata) {
             val joinResult = JoinGroupResult(
               members = if (group.isLeader(member.memberId)) {
+                info(s"Reply the group leader with the following dynamic members' metadata: ${group.allDynamicMembers}" +
+                  s"and static members' metadata: ${group.allStaticMembers}")
```

Comment:
Shouldn't we include the group id in the log as well?

Reply:
Good idea